Assets/MixedRealityToolkit/Services/BaseDataProviderAccessCoreSystem.cs

```diff
@@ -19,13 +19,23 @@ public abstract class BaseDataProviderAccessCoreSystem : BaseCoreSystem, IMixedR
     {
         private List<IMixedRealityDataProvider> dataProviders = new List<IMixedRealityDataProvider>();
 
+        protected bool isDestroyed = false;
```

Comment:
You mention this is a partial fix. Would a test to change MRTK.ActiveProfile possibly succeed?

Reply:
yes. it would